main/src/cgeo/geocaching/CacheDetailActivity.java

```diff
@@ -2294,11 +2297,6 @@ protected Void doInBackground(final Void... params) {
                 DataStore.saveCache(cache, EnumSet.of(SaveFlag.DB));
                 return null;
             }
-
-            @Override
-            protected void onPostExecute(final Void v) {
-                notifyDataSetChanged();
-            }
```

Comment:
Yes. The detail activity must know when the data was changed and when it has to update the UI.